pkg/v2/system.go

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2023 Seagate Technology LLC and/or its Affiliates
+
+package mcapi
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/Seagate/seagate-exos-x-api-go/pkg/common"
+	"k8s.io/klog/v2"
+)
+
+//
+// System Information Storage and Creation
+//
+
+// systems: Data object storing all system information for all added controllers
+var systems common.SystemsData
+
+// AddSystem: Uses the client to query and store system data
+func AddSystem(url string, client *Client) error {
```

Comment:
Maybe add a comment suggestion how this would be expanded to support IP addresses for both controllers?

Reply:
Added a simple comment, but we will need to discuss a broader design change.